cmd/stack.go

```diff
@@ -627,7 +627,7 @@ func init() {
 	tui.CheckErr(AddOptions(stackUpdateCmd, false))
 
 	// Delete Stack (Down)
-	stackCmd.AddCommand(tui.AddDependencyCheck(stackDeleteCmd, tui.Pulumi))
+	stackCmd.AddCommand(tui.AddDependencyCheck(stackDeleteCmd, tui.Pulumi, tui.Docker))
```

Comment:
Now that I think about this with the release of our terraform providers we'll want to shift more of this to our providers, possibly implementing somekind of preflight check for dependencies, so our terraform users don't require pulumi to be installed simply to execute the commands.

Reply:
Created an issue here: https://github.com/nitrictech/nitric/issues/649